docs/spl-docs/makemv-command.md

```diff
@@ -0,0 +1,96 @@
+# makemv
+
+## Description
+
+Converts a single valued field into a multivalue field by splitting the values on a string delimiter or by using a regular expression. The delimiter can be a multicharacter delimiter.
+
+## Syntax
+
+`makemv [delim=<string> | tokenizer=<string>] [allowempty=<bool>] [setsv=<bool>] <field>`
+
+## Required Arguments
+
+### field
+
+**Syntax**: `<field>` \
+**Description**: The name of a field to generate the multivalues from.
+
+## Optional Arguments
+
+### delim
+
+**Syntax**: `delim=<string>` \
+**Description**: A string value used as a delimiter. Splits the values in `field` on every occurrence of this delimiter. \
+**Default**: A single space (" ").
```

Comment:
I think `delim` can be multiple characters, right? I don't think it ever came up in your examples, so it would be good to mention that somewhere (and also whether it would split on any of those characters, or only split on that exact sequence)